pkg/kubelet/cm/cpumanager/cpu_manager.go

```diff
@@ -42,7 +42,7 @@ import (
 type ActivePodsFunc func() []*v1.Pod
 
 type runtimeService interface {
-	UpdateContainerResources(id string, resources *runtimeapi.LinuxContainerResources) error
```

Comment:
not a blocker but rather just to confirm, changing this parameter from `LinuxContainerResources` to `ContainerResources` could be decoupled from the CRI changes right? remote_runtime would just need to call UpdateContainerResources with LinuxContainerResources directly (windows would not be supported tho).

Reply:
Does runtime call this? IIUC, runtime responds to it, and yes it does not have to change. The UpdateContainerResourcesRequest.Linux field will contain what it needs same as before. Windows runtime looks at UpdateContainerResourcesRequest.Windows.

Reply:
windows was already there.. just wasn't being used/set here

 because of the way the changes went in (not changing the CRI api except for the additional status field and some additional context) the client side manager proto should only impact cri-tools implementing the manager and container runtimes doing integration by implementing a manager for test

Reply:
I'd be good with decoupling the client side as well.. if desired..

Reply:
noting windows was there in the request before, but was not being used:
message UpdateContainerResourcesRequest {
     // ID of the container to update.
     string container_id = 1;
    // Resource configuration specific to Linux containers.
    LinuxContainerResources linux = 2;
    // Resource configuration specific to Windows containers.
    WindowsContainerResources windows = 3;
    // Unstructured key-value map holding arbitrary additional information for
    // container resources updating. This can be used for specifying experimental
    // resources to update or other options to use when updating the container.
    map<string, string> annotations = 4;
}

Reply:
> IIUC, runtime responds to it.

That I understand. I was just trying to confirm the parameters here did not need to change.

> the client side manager proto should only impact cri-tools implementing the manager and container runtimes doing integration by implementing a manager for test

Gotcha. Yeah in that case this should have very little risk. I am okay with leaving this as the current PR is.